modules/cli/pages/podman.adoc

```diff
@@ -0,0 +1,242 @@
+Skupper is enabled for "<username>" with site name "<machine-name>-<username>" in interior mode. It is not connected to any other sites. It has no exposed services.
+----
+
+NOTE: You can only create one site per user. If your require a host to support many sites, create a user for each site.
+
+--
+
+
+== Linking sites using Skupper podman
+
+The general flow for linking podman sites is the same as for Kubernetes sites:
+
+. Generate a token on one site:
++
+----
+$ skupper token create <filename>
+----
+
+. Create a link from the other site:
++
+----
+$ skupper link create <filename>
+----
+
+After you have linked to a network, you can check the link status:
+
+----
+$ skupper link status
+----
+
+
+== Working with services using Skupper podman
+
+The general flow for working with services is the same for Kubernetes and Podman sites.
+However, consider the following:
+
+* Services exposed on Kubernetes are not automatically available to Podman sites.
+This is the equivalent to Kubernetes sites created using `skupper init --enable-service-sync false`.
+* On Kubernetes sites, `skupper service status` shows details about original service location.
+This information might not be available from Podman sites.
+
+.Example 01: Consuming a service from a Podman site
+
+In this variation of the link:https://github.com/skupperproject/skupper-example-hello-world[hello world] example, the `backend` service is exposed on Kubernetes site and a Podman site is linked.
+You deploy the `frontend` as a container and that container can access the `backend` service.
+
+. Create a Podman site and link it to a Kubernetes site.
+
+. Check the service from the Podman site:
++
+----
+$ skupper service status
+
+No services defined
+----
++
+This result is expected because services exposed on Kubernetes are not automatically available to Podman sites.
+
+. Create a service on the Podman site matching the service exposed on the Kubernetes site:
++
+----
+$ skupper service create backend 8080
+----
+
+. Validate the service from the Podman site by checking the backend API health URL:
++
+--
+----
+$ podman run -it --network=skupper --name=myubi ubi8/ubi curl backend:8080/api/health
+
+OK
+----
+
+This command runs a container using the `skupper` network and returns the results from `http://backend:8080/api/health`
+--
+
+. Run the frontend as a container:
++
+----
+$ podman run -dp 8080:8080 --name hello-world-frontend --network skupper quay.io/skupper/hello-world-frontend
+----
+
+. Check your {service-network} is working as expected by navigating to http://localhost:8080 and click *Say hello*.
++
+--
+Each of the backend replicas respond, for example `Hi, Perfect Parrot. I am Kind Hearted Component (backend-7c84887f9f-wxhxp).`
+
+[NOTE]
+====
+In this scenario, running the `skupper service status` command on the Podman site does not provide much detail about the service:
+
+----
+$ skupper service status
+Services exposed through Skupper:
+╰─ backend (tcp port 8080)
```

Comment:
I believe this output is the same you'd get on a Kubernetes site,
for a remote service. 

The target information is only displayed when a given service is bound
to target resources.

Reply:
agreed